code/modules/reagents/chemistry/reagents/misc.dm

```diff
@@ -464,12 +464,12 @@
 	C.AddElement(/datum/element/waddling)
 
 /datum/reagent/jestosterone/on_mob_life(mob/living/carbon/M)
-	if(!istype(M))
-		return ..()
 	var/update_flags = STATUS_UPDATE_NONE
 	if(prob(10))
 		M.emote("giggle")
-	if(M?.mind.assigned_role == "Clown")
+	if(!M.mind)
+		return ..() | update_flags
+	if(M.mind.assigned_role == "Clown")
```

Comment:
This does change behaviour. Namely, mindless mobs now are not affected by this drug other than that they giggle occasionally.

Reply:
I dont think anyone has ever tried to use jestosterone on a mindless mob, nor would there be much point in it.

PR is still inside fix territory.